example/android/app/build.gradle

```diff
@@ -100,6 +100,7 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion rootProject.ext.compileSdkVersion
+    ndkVersion '20.1.5948944'
```

Comment:
This should fix an error in CircleCI:

`No version of NDK matched the requested version 21.0.6113669. Versions available locally: 20.1.5948944`